.circleci/config.yml

```diff
@@ -2,17 +2,14 @@ version: 2
 jobs:
   build:
     docker:
-      - image: cimg/node:12.22.12  # April 2022
+      - image: cimg/node:16.15
     steps:
       - checkout
-      - run:
-          name: Update npm
-          command: 'npm install -g npm@latest'
       - restore_cache:
           key: dependency-cache-{{ checksum "package.json" }}
       - run:
           name: Install Dependencies
-          command: npm install
+          command: npm ci --force
```

Comment:
why force?

Reply:
so `npm ci` revealed an issue in our dependency tree:

```
    "rollup": "^2.3.3",
    "rollup-plugin-babel": "^4.4.0",
    "rollup-plugin-commonjs": "^10.1.0",
    "rollup-plugin-node-resolve": "^5.2.0",
    "rollup-plugin-uglify": "^6.0.4",
```
    
`rollup-plugin-uglify` on the _latest_ version (6.0.4) expects a version of `rollup` < 2. we are using 2.3.3. the upgrade was an attempt to see if they had updated the rules.

given that these are dev dependencies and the commands appear to working, I added the `--force` flag to ignore that check.

the other option is to downgrade rollup, which is not great, either.